app/controllers/AnnotationIOController.scala

```diff
@@ -281,10 +281,19 @@ Expects:
           .map(layer => elementClassToProto(layer.elementClass))
           .getOrElse(elementClassToProto(VolumeTracingDefaults.elementClass)),
         fallbackLayer = fallbackLayer.map(_.name),
-        largestSegmentId = fallbackLayer.map(_.largestSegmentId).getOrElse(VolumeTracingDefaults.largestSegmentId)
+        largestSegmentId =
+          combineLargestSegmentIds(volumeTracing.largestSegmentId, fallbackLayer.map(_.largestSegmentId))
```

Comment:
`volumeTracing.largestSegmentId` is from the loaded nml and has precedence now, is this correct? It's not clear to me how `adaptPropertiesToFallbackLayer` is used, but probably fine.